daffodil-test-ibm1/src/test/resources/test-suite/tresys-contributed/AG.tdml

```diff
@@ -24,9 +24,9 @@
   <parserTestCase name="AG000" root="allZones" model="AG.dfdl.xsd"
     description="Discriminated choices"
     roundTrip="true">
-    <document><![CDATA[1
-zone 1
-]]></document>
+    <document>
+	  <documentPart type="text" replaceDFDLEntities="true"><![CDATA[1%LF;zone 1%LF;]]></documentPart>
```

Comment:
I wonder how many tests would break if we changed the default for replaceDFDLEntitites so it defaults to "true". That would make these less cluttered, by allowing just <document><![CDATA[...]]></document> again.

Reply:
I imagine anything with a percent sign would break since that would look like an invalid DFDL entitity? A quick grep shows about 65 lines that contain "document" and a percent and don't already have replaceDFDLEntities set. So I'd guess at least that many would need replaceDFDLEntities="false", which is probably about the same amount that needed to be changed in this commit so I'm not sure it would be all the different.